qiskit_ibm_runtime/runtime_job.py

```diff
@@ -655,3 +656,24 @@ def tags(self) -> List:
             Tags assigned to the job that can be used for filtering.
         """
         return self._tags
+
+    @property
+    def cost_estimation(self) -> Dict[str, Any]:
```

Comment:
Can we not call this `cost_estimation`? Since the returned values are estimated execution time, which is at best an indirect translation to cost. Maybe `estimated_execution_time`?

Reply:
Sure, @francabrera fyi

Reply:
since this is an estimation of `usage.quantum_seconds` let's make it `usage_estimation.quantum_seconds` the value being `estimated_running_time_seconds` cc @jyu00